src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tableRows = table.rows;
+
+for (const row of tableRows) {
+  const columnPosition = row.cells[1];
+
+  const columnPositionCopy = columnPosition.cloneNode(true);
+
+  const columnSalary = row.cells[4];
+
+  columnSalary.remove();
+
+  row.append(columnPositionCopy);
+
+  row.append(columnSalary);
+}
```

Comment:
You can not separate lines with the same logic with an empty line, separate blocks with different logic (declaration of variables, calculations, return value from a function, etc.)
```suggestion
for (const row of tableRows) {
  const columnPosition = row.cells[1];
  const columnPositionCopy = columnPosition.cloneNode(true);
  const columnSalary = row.cells[4];

  columnSalary.remove();
  row.append(columnPositionCopy);
  row.append(columnSalary);
}
```